Index: dpdk-18.11/config/common_linuxapp
===================================================================
--- dpdk-18.11.orig/config/common_linuxapp
+++ dpdk-18.11/config/common_linuxapp
@@ -22,7 +22,7 @@ CONFIG_RTE_LIBRTE_PMD_SOFTNIC=y
 CONFIG_RTE_LIBRTE_PMD_TAP=y
 CONFIG_RTE_LIBRTE_AVP_PMD=y
 CONFIG_RTE_LIBRTE_VDEV_NETVSC_PMD=y
-CONFIG_RTE_LIBRTE_NFP_PMD=y
+CONFIG_RTE_LIBRTE_NFP_PMD=n
 CONFIG_RTE_LIBRTE_POWER=y
 CONFIG_RTE_VIRTIO_USER=y
 CONFIG_RTE_PROC_INFO=y
@@ -33,26 +33,26 @@ CONFIG_RTE_LIBRTE_NETVSC_PMD=y
 #
 # Common libraries, before Bus/PMDs
 #
-CONFIG_RTE_LIBRTE_COMMON_DPAAX=y
+CONFIG_RTE_LIBRTE_COMMON_DPAAX=n
 
 # NXP DPAA BUS and drivers
-CONFIG_RTE_LIBRTE_DPAA_BUS=y
-CONFIG_RTE_LIBRTE_DPAA_MEMPOOL=y
-CONFIG_RTE_LIBRTE_DPAA_PMD=y
-CONFIG_RTE_LIBRTE_PMD_DPAA_EVENTDEV=y
-CONFIG_RTE_LIBRTE_PMD_DPAA_SEC=y
-CONFIG_RTE_LIBRTE_PMD_CAAM_JR=y
+CONFIG_RTE_LIBRTE_DPAA_BUS=n
+CONFIG_RTE_LIBRTE_DPAA_MEMPOOL=n
+CONFIG_RTE_LIBRTE_DPAA_PMD=n
+CONFIG_RTE_LIBRTE_PMD_DPAA_EVENTDEV=n
+CONFIG_RTE_LIBRTE_PMD_DPAA_SEC=n
+CONFIG_RTE_LIBRTE_PMD_CAAM_JR=n
 
 # NXP FSLMC BUS and DPAA2 drivers
-CONFIG_RTE_LIBRTE_FSLMC_BUS=y
-CONFIG_RTE_LIBRTE_DPAA2_MEMPOOL=y
-CONFIG_RTE_LIBRTE_DPAA2_PMD=y
-CONFIG_RTE_LIBRTE_PMD_DPAA2_EVENTDEV=y
-CONFIG_RTE_LIBRTE_PMD_DPAA2_SEC=y
-CONFIG_RTE_LIBRTE_PMD_DPAA2_CMDIF_RAWDEV=y
-CONFIG_RTE_LIBRTE_PMD_DPAA2_QDMA_RAWDEV=y
+CONFIG_RTE_LIBRTE_FSLMC_BUS=n
+CONFIG_RTE_LIBRTE_DPAA2_MEMPOOL=n
+CONFIG_RTE_LIBRTE_DPAA2_PMD=n
+CONFIG_RTE_LIBRTE_PMD_DPAA2_EVENTDEV=n
+CONFIG_RTE_LIBRTE_PMD_DPAA2_SEC=n
+CONFIG_RTE_LIBRTE_PMD_DPAA2_CMDIF_RAWDEV=n
+CONFIG_RTE_LIBRTE_PMD_DPAA2_QDMA_RAWDEV=n
 
 #
 # NXP ENETC PMD Driver
 #
-CONFIG_RTE_LIBRTE_ENETC_PMD=y
+CONFIG_RTE_LIBRTE_ENETC_PMD=n
Index: dpdk-18.11/kernel/linux/igb_uio/Makefile
===================================================================
--- dpdk-18.11.orig/kernel/linux/igb_uio/Makefile
+++ dpdk-18.11/kernel/linux/igb_uio/Makefile
@@ -14,7 +14,7 @@ MODULE_PATH = drivers/net/igb_uio
 #
 MODULE_CFLAGS += -I$(SRCDIR) --param max-inline-insns-single=100
 MODULE_CFLAGS += -I$(RTE_OUTPUT)/include
-MODULE_CFLAGS += -Winline -Wall -Werror
+MODULE_CFLAGS += -Winline -Wall -Werror -Wno-implicit-fallthrough
 MODULE_CFLAGS += -include $(RTE_OUTPUT)/include/rte_config.h
 
 #
Index: dpdk-18.11/config/common_base
===================================================================
--- dpdk-18.11.orig/config/common_base
+++ dpdk-18.11/config/common_base
@@ -38,7 +38,7 @@ CONFIG_RTE_ARCH_STRICT_ALIGN=n
 #
 # Compile to share library
 #
-CONFIG_RTE_BUILD_SHARED_LIB=n
+CONFIG_RTE_BUILD_SHARED_LIB=y
 
 #
 # Use newest code breaking previous ABI
@@ -202,13 +202,13 @@ CONFIG_RTE_LIBRTE_BNXT_PMD=y
 #
 # Compile burst-oriented Chelsio Terminator (CXGBE) PMD
 #
-CONFIG_RTE_LIBRTE_CXGBE_PMD=y
+CONFIG_RTE_LIBRTE_CXGBE_PMD=n
 CONFIG_RTE_LIBRTE_CXGBE_DEBUG=n
 CONFIG_RTE_LIBRTE_CXGBE_DEBUG_REG=n
 CONFIG_RTE_LIBRTE_CXGBE_DEBUG_MBOX=n
 CONFIG_RTE_LIBRTE_CXGBE_DEBUG_TX=n
 CONFIG_RTE_LIBRTE_CXGBE_DEBUG_RX=n
-CONFIG_RTE_LIBRTE_CXGBE_TPUT=y
+CONFIG_RTE_LIBRTE_CXGBE_TPUT=n
 
 # NXP DPAA Bus
 CONFIG_RTE_LIBRTE_DPAA_BUS=n
Index: dpdk-18.11/mk/rte.lib.mk
===================================================================
--- dpdk-18.11.orig/mk/rte.lib.mk
+++ dpdk-18.11/mk/rte.lib.mk
@@ -55,7 +55,7 @@ exe2cmd = $(strip $(call dotfile,$(patsu
 
 ifeq ($(LINK_USING_CC),1)
 # Override the definition of LD here, since we're linking with CC
-LD := $(CC) $(CPU_CFLAGS) $(EXTRA_CFLAGS)
+override LD := $(CC) $(CPU_CFLAGS) $(EXTRA_CFLAGS)
 _CPU_LDFLAGS := $(call linkerprefix,$(CPU_LDFLAGS))
 override EXTRA_LDFLAGS := $(call linkerprefix,$(EXTRA_LDFLAGS))
 else
